classes/controller/class-enform-fields-list-table.php

```diff
@@ -100,14 +100,15 @@ public function get_columns() {
 	 *
 	 * @return string Content for column.
 	 */
-	protected function column_default( $item, $column_name ) {
+	protected function column_default( $item, $column_name ): ?string {
 		switch ( $column_name ) {
 			case 'id':
 			case 'name':
 			case 'property':
 			case 'tag':
 				return $item[ $column_name ];
 		}
+		return '';
```

Comment:
You can make this return `null` instead explicitly, then the return value will be like before.